tests/unit/amo/utils/test_index.js

```diff
@@ -205,6 +205,19 @@ describe(__filename, () => {
           }).isInternal,
         ).toEqual(false);
       });
+
+      it('returns false for a blog URL containing the current host', () => {
+        const baseURL = 'https://example.org';
+        const urlString = url.format({
+          ...url.parse(baseURL),
+          pathname: '/blog/some-blog-post/',
+        });
```

Comment:
Is this the same as...?

```js
const urlString = `${baseURL}/blog/some-blog-post/`;
```

Reply:
Pretty much. I just copied an existing test and changed it. At least it maintains some consistency across the tests. 😉

Reply:
It isn't really clear what's going on with `url.format()` and `pathname`, though. I think having a simple template literal would be a bit better.

Reply:
Sure, I can change that.